remediation/machine-deletion-remediation/machine-deletion-remediation.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Machine Deletion Remediation
+nav_order: 3
+has_children: true
+parent: Remediation
+---
+
+# Machine-API Driven Remediation
```

Comment:
Why not just `Machine Deletion Remediation`?

Reply:
I didn't change this specific line, but I agree it makes sense to change it